cmake/modules/FindOpenSSL.cmake

```diff
@@ -0,0 +1,37 @@
+# - Find OpenSSL Header and Libraries
+#
+# OPENSSL_PREFIX - where to find ssl.h and libraries
+# OPENSSL_FOUND - True if found.
+
+set(OPENSSL_LIB_DIR "${OPENSSL_PREFIX}/lib")
+
+find_path(OPENSSL_INCLUDE_DIR ssl/ssl.h NO_DEFAULT_PATH PATHS
+  /usr/include
+  /opt/local/include
+  /usr/local/include
+  "${OPENSSL_PREFIX}/include"
+  )
+
+find_library(LIBSSL NAMES ssl)
```

Comment:
setting openssl_libs would improve things;  on the simplification, I don't think it's helpful, really